website/docs/r/cognito_identity_pool_roles_attachment.markdown

```diff
@@ -99,7 +99,7 @@ resource "aws_cognito_identity_pool_roles_attachment" "main" {
 
 The Cognito Identity Pool Roles Attachment argument layout is a structure composed of several sub-resources - these resources are laid out below.
 
-* `identity_pool_id` (Required) - An identity pool ID in the format REGION:GUID.
+* `identity_pool_id` (Required) - An identity pool ID in the format `REGION_GUID`.
```

Comment:
Looking at the provider declaration, it seems this resource [calls SetIdentityPoolRoles](https://github.com/hashicorp/terraform-provider-aws/blob/78b2dea023f65b5644f783663f32942227e16966/internal/service/cognitoidentity/pool_roles_attachment.go#L134). Looking at the AWS Go SDK for [that function's input](https://docs.aws.amazon.com/sdk-for-go/api/service/cognitoidentity/#SetIdentityPoolRolesInput), it seems to indicate the correct format is `REGION:GUID`. Are you seeing differing behavior in practice?